Libraries/LibGfx/TextLayout.cpp

```diff
@@ -24,7 +24,22 @@ RefPtr<GlyphRun> shape_text(FloatPoint baseline_start, float letter_spacing, Utf
     Vector<hb_glyph_info_t> const input_glyph_info({ glyph_info, glyph_count });
 
     auto* hb_font = font.harfbuzz_font();
-    hb_shape(hb_font, buffer, nullptr, 0);
+    hb_feature_t const* hb_features_data = nullptr;
+    Vector<hb_feature_t> hb_features;
+    if (!features.is_empty()) {
+        hb_features.ensure_capacity(features.size());
+        for (auto const& feature : features) {
+            hb_features.append({
+                .tag = HB_TAG(feature.tag[0], feature.tag[1], feature.tag[2], feature.tag[3]),
+                .value = feature.value,
+                .start = 0,
+                .end = static_cast<unsigned int>(-1),
```

Comment:
Are you looking for `NumericLimits<u32>::max()` by any chance?

Reply:
Nope, but HB_FEATURE_GLOBAL_END is better!